.github/workflows/filename_workflow.yml

```diff
@@ -0,0 +1,46 @@
+name: aws-api-filename-test
+on:
+  push:
+    branches:
```

Comment:
Maybe we could just keep `workflow_dispatch` and remove this?

Reply:
@linglp this was just for getting the remote run results and testing the workflow before this PR is merged. It will be removed before merge